src/Account.js

```diff
@@ -147,7 +183,6 @@ class Accounts {
         try {
             address = this.Wallet.Utils.isAddress(address)
             let accountIndex = await this._getAccountIndex(address)
-            this.accounts[accountIndex]["UTXO"] = { "DataStores": [], "ValueStores": [], "AtomicSwaps": [], "ValueStoreIDs": [], "DataStoreIDs": [], "AtomicSwapIDs": [], "Value": "" }
```

Comment:
Why are we removing this? , iirc it clears the UTXO state before updating it below so it's accurate